CMakeLists.txt

```diff
@@ -81,6 +83,8 @@ endif()
 target_include_directories(${binary} SYSTEM PRIVATE ${X11_INCLUDE_DIRS})
 target_link_libraries(${binary} PRIVATE ${X11_LIBRARIES})
 
+target_link_libraries(${binary} PRIVATE pulse pulse-simple fdk-aac)
```

Comment:
Could we have this as a submodules like the other non-system dependencies?

Reply:
Thats for the `fdk-aac` btw. I would class `pulse` as a system library. Not sure how to find it in Cmake

Reply:
fdk-aac has its own license, gotta look into that

Reply:
Its got a BSD license

Reply:
Then let's add it as submodule.